src/db/mod.rs

```diff
@@ -0,0 +1,25 @@
+use tokio_postgres::{Client, NoTls, Error};
+use dotenv::dotenv;
+use std::env;
+use tokio::sync::OnceCell as TokOnceCell;
```

Comment:
Why are we renaming this?

Reply:
It was just in the docs, can use just as it is tho.